libs/miranum-core/src/lib/miranum-core.ts

```diff
@@ -61,10 +61,12 @@ export class MiranumCore {
          */
         let pathInProject = undefined;
         const lastFolder = projectPath.substring(projectPath.lastIndexOf("/")+1);
-        if(this.projectConfig && lastFolder === projectName) {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        if(this.projectConfig?.workspace[this.camelize(`${type}s`)] && lastFolder === projectName) {
```

Comment:
In der generate Domain fehlen aktuell noch die Dateiendungen. Wenn die Config gesetzt ist sollen die Dateiendungen aus der Config genommen werden, sonst ein default.